packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -768,6 +775,9 @@ class PopupMenuButton<T> extends StatefulWidget {
   /// Called when the user selects a value from the popup menu created by this button.
   final PopupMenuItemSelected<T> onSelected;
 
+  /// Called when the user dismisses the popup menu without selecting an item.
+  final PopupMenuCancelled<T> onCancelled;
```

Comment:
maybe mention onSelected from onCancelled and onCancelled from onSelected

Reply:
Done...let me know if I did it how you were thinking.